specification/2.0/Specification.adoc

```diff
@@ -1064,13 +1274,9 @@ The following example extends the Mesh defined in the previous example to a morp
 }
 ----
 
-After applying morph targets to vertex positions and normals, tangent space may need to be recalculated. See <<appendix-a-tangent-space-recalculation,Appendix A>> for details.
+The number of morph targets is not limited. Client implementations **SHOULD** support at least eight morphed attributes. This means that they **SHOULD** support at least eight morph targets that contain a `POSITION` attribute, or four morph targets that contain a `POSITION` and a `NORMAL` attribute, or two morph targets that contain `POSITION`, `NORMAL` and `TANGENT` attributes.
```

Comment:
I see that this previously was an implementation note. But now, it made me (honestly) wonder whether an implementation SHOULD also support eight morph targets that (only) contain a NORMAL attribute. 

I think this could be avoided by more generically (and elaborately) saying that the implementation

> ...SHOULD support eight morph targets when each morph target has one attribute, four morph targets where each morph target has two attributes, or two morph targets where each morph target has three or four attributes